doc/rel_notes/0.5.1.txt

```diff
@@ -27,3 +27,7 @@ Documentation
    added algorithms and containers and sorted the lists.
 
   * Added missing algorithm on the OCV module Doxygen main page and sorted.
+
+OCV Plugin
+
+  * Flipped repective returns in width/height getter methods.
```

Comment:
[nit-pick] Might want to clarify that they were flipped because they were previously incorrect.